test/k8s_logging_tests/test_config_logging.py

```diff
@@ -81,3 +81,24 @@ def test_metric_index_from_annotations(setup, index, expected):
     logger.info("Splunk received %s events in the last minute",
                 len(events))
     assert len(events) >= expected
+
+@pytest.mark.parametrize("index,sourcetype,expected", [
+    ("test_metrics", "sourcetype-anno", 1)
+])
+def test_metric_sourcetype_from_annotations(setup, index, sourcetype, expected):
+
+    '''
+    Test that metrics are being assigned the "sourcetype-anno" sourcetype, as defined by splunk.com/sourcetype annotation added during setup
+    '''
+    logger.info("testing for metrics index={0} sourcetype={1} expected={2} event(s)".format(index, sourcetype, expected))
+    search_query = "index={0} filter=\"sourcetype={1}\"".format(index, sourcetype)
+
+    events = check_events_from_splunk(start_time="-1h@h",
+                                      url=setup["splunkd_url"],
+                                      user=setup["splunk_user"],
+                                      query=["mpreview {0}".format(
+                                          search_query)],
+                                      password=setup["splunk_password"])
+    logger.info("Splunk received %s events in the last minute",
+                len(events))
+    assert len(events) >= expected
```

Comment:
This test is failing in CI/CD still, see what you can do about it.
- You might need to add the `splunk.com/sourcetype: "sourcetype-anno"` annotation right below here. https://github.com/signalfx/splunk-otel-collector-chart/blob/ae4b01e8b3e2ffeaea404994e8b002e3a697ac61/test/test_setup.yaml#L28